backend/sirius-web-spring-collaborative-forms/src/main/java/org/eclipse/sirius/web/spring/collaborative/forms/FormEventProcessor.java

```diff
@@ -108,25 +109,27 @@ public ISubscriptionManager getSubscriptionManager() {
 
     @Override
     public Optional<EventHandlerResponse> handle(IRepresentationInput representationInput, Context context) {
+        Optional<EventHandlerResponse> result = Optional.empty();
         if (representationInput instanceof IFormInput) {
             IFormInput formInput = (IFormInput) representationInput;
 
             if (formInput instanceof UpdateWidgetFocusInput) {
                 UpdateWidgetFocusInput input = (UpdateWidgetFocusInput) formInput;
                 this.widgetSubscriptionManager.handle(input, context);
+                result = Optional.of(new EventHandlerResponse(true, representation -> true, new UpdateWidgetFocusSuccessPayload(input)));
```

Comment:
The constructor arguments are wrong. Changing the focus does *not* make the editing context dirty (first arg), and does *not* require all representations to be refreshed (second arg).

Reply:
Oh right. I fixed it.

Reply:
Still wrong, the latest version passes: `true, representation -> input.getRepresentationId().equals(representation.getId())`.

It should be `false` and `() -> false`.
* The first argument indicates if the "semantic model" has been made dirty by the operation (`true` if dirty). Setting the focus on a different widget does *not* modify the semantic model.
* The second argument is a function which detemines for each representation it it should be refreshed. Again, even for the current representation/form, changing the focus does *not* modify its state and there is no reason to launch a refresh on it.

Reply:
I fixed it in the new PR (https://github.com/eclipse-sirius/sirius-components/pull/204) as this one is on my fork.